### PR TITLE
Add a doctest to hashes/hamming_code.py

### PR DESCRIPTION
**Pull request by tielarose**
### Describe your change:

Added a doctest to hashes/hamming_code.py to increase test coverage to 100%

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [X] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [X] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by tielarose** [comment]
I see that the build is failing with this error:

=========================== short test summary info ============================
FAILED web_programming/fetch_anime_and_[play.py](http://play.py/)::web_programming.fetch_anime_and_play.get_anime_episode
============ 1 failed, 1725 passed, 1 warning in 114.91s (0:01:54) =============
Error: Process completed with exit code 1.

But I hadn't touched that file at all. Should I be trying to figure out what's wrong in that file? Even if I could figure it out, wouldn't that mean my pull request would touch more than one algorithm, which we're not supposed to do?

File: hashes/hamming_code.py [hunk 1]
```diff
@@ -77,6 +77,10 @@ def emitter_converter(size_par, data):
 
     >>> emitter_converter(4, "101010111111")
     ['1', '1', '1', '1', '0', '1', '0', '0', '1', '0', '1', '1', '1', '1', '1', '1']
+    >>> emitter_converter(5, "101010111111")
```
Comment (imSanko) [inline hashes/hamming_code.py:80]: Have you tried running `python -m doctest -v hashes/hamming_code.py`

**Review by imSanko — CHANGES_REQUESTED** [review]

**Comment by imSanko** [comment]
> I see that the build is failing with this error:
> 
> =========================== short test summary info ============================ FAILED web_programming/fetch_anime_and_[play.py](http://play.py/)::web_programming.fetch_anime_and_play.get_anime_episode ============ 1 failed, 1725 passed, 1 warning in 114.91s (0:01:54) ============= Error: Process completed with exit code 1.
> 
> But I hadn't touched that file at all. Should I be trying to figure out what's wrong in that file? Even if I could figure it out, wouldn't that mean my pull request would touch more than one algorithm, which we're not supposed to do?

We need to check with it
**********************************************************************
2 items had failures:
   1 of   1 in hamming_code.emitter_converter
   1 of   1 in hamming_code.receptor_converter
4 tests in 5 items.
2 passed and 2 failed.
***Test Failed*** 2 failures.

File: hashes/hamming_code.py (hunk 1 above)
Comment (tielarose) [reply hashes/hamming_code.py:80]: I did, when I run it it shows this:

1 items had no tests:
    hamming_code
4 items passed all tests:
   1 tests in hamming_code.emitter_converter
   1 tests in hamming_code.receptor_converter
   1 tests in hamming_code.text_from_bits
   1 tests in hamming_code.text_to_bits
4 tests in 5 items.
4 passed and 0 failed.
Test passed.

But I see below that when you ran it there were tests failing in this file. Any guidance on what I might be doing wrong that tests are passing for me, but not when someone else is running it? Thank you for your help! This is my first attempt at an open source contribution so I truly appreciate your help navigating all of this :)

**Comment by cclauss** [comment]
Looking at the failed `build` tests, it was some other file that failed.  This one looks like it correctly tested the exceptions.

If the rerun of the tests pass then let's merge this one.

**Review by cclauss — APPROVED** [review]

**Pull request merged**